app/client/src/config/profiles.ts

```diff
@@ -2,232 +2,431 @@ export const profiles = {
   actions: {
     description:
       'Contains detailed information on plans to restore and protect water quality including Total Maximum Daily Loads (TMDLs), 4b Plans, 5R Plans, Protection Approaches, and other plans.',
-    columns: new Set([
-      'objectId',
-      'actionAgency',
-      'actionId',
-      'actionName',
-      'actionType',
-      'assessmentUnitId',
-      'assessmentUnitName',
-      'completionDate',
-      'includeInMeasure',
-      'inIndianCountry',
-      'locationDescription',
-      'organizationId',
-      'organizationName',
-      'organizationType',
-      'parameter',
-      'region',
-      'state',
-      'waterSize',
-      'waterSizeUnits',
-      'waterType',
+    columns: new Map<string, { contextColumns?: string[] }>([
+      ['objectId', {}],
+      ['actionAgency', {}],
+      [
+        'actionId',
+        {
+          contextColumns: [
+            'actionName',
+            'organizationId',
+            'organizationName',
+            'state',
```

Comment:
I'm sure we can do that. At the last meeting, the team said they were really only worried about `region` filtering `state`, but since they're in the same MV group then it shouldn't hurt.